client/src/hooks/useFriends.js

```diff
@@ -0,0 +1,12 @@
+import { useEffect, useState } from "react";
+import { getItemsFromLocalStorage } from "../utils/itemStorage";
+
+export default function useFriends() {
+  const [friends, setFriends] = useState([]);
+
+  useEffect(() => {
+    const friends = getItemsFromLocalStorage("friends");
+    setFriends(friends);
+  }, []);
+  return [friends, setFriends];
```

Comment:
I think you could use your `itemStorage` functions (the ones related to `friends` at least) in here to make the hook more powerful